CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `SyncOptions::{account, alias, nft}` fields;
 - `{TransactionOptions, TransactionOptionsDto}::burn`;
 - `Memory` storage adapter;
+- Coverall github action;
```

Comment:
I think this changelog is only for the crate itself :trollface:

Reply:
Yeah no need for that